inference-engine/tests/functional/plugin/cpu/test_utils/cpu_test_utils.cpp

```diff
@@ -54,14 +58,16 @@ std::string CPUTestsBase::impls2str(const std::vector<std::string> &priority) {
     for (auto &impl : priority) {
         ((str += "cpu:") += impl) += ",";
     }
-    str.erase(str.end() - 1);
+    if (!str.empty()) {
+        str.pop_back();
+    }
     return str;
 }
 
-void CPUTestsBase::CheckCPUImpl(InferenceEngine::ExecutableNetwork &execNet, std::string nodeType,
-                                std::vector<cpu_memory_format_t> inputMemoryFormats,
-                                std::vector<cpu_memory_format_t> outputMemoryFormats, std::string selectedType) {
+void CPUTestsBase::CheckCPUImpl(InferenceEngine::ExecutableNetwork &execNet, std::string nodeType) const {
     IE_SUPPRESS_DEPRECATED_START
+    ASSERT_TRUE(!selectedType.empty()) << "Node type is not defined.";
```

Comment:
BTW. As I remember `_BF16` part in primitive type is defined by input precision[s]. In that case we don't check output precision at all, but it seems to be important.

Reply:
I agree, but the solution lies outside the scope of this PR. It may be worth to discuss this topic separately.

Reply:
Lets create dedicate ticket in that case.

Reply:
Dedicated ticket has been created.